modules/nw-install-contour-operator.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * networking/about-contour-operator.adoc
+
+[id="nw-install-contour-operator_{context}"]
+= Installing Contour Operator
+
+Install Contour Operator on {product-title} to use link:https://gateway-api.sigs.k8s.io/[Gateway API].
+
+.Prerequisites
+
+* You installed an {product-title} cluster and the `oc` command line.
+//* Log in as a user with cluster-admin permission.
+
+.Procedure
+
+. Install Contour Operator:
++
+[source,terminal]
+----
+$ oc apply -f https://raw.githubusercontent.com/projectcontour/contour-operator/v1.16.0/examples/operator/operator.yaml
+----
++
+[NOTE]
+====
+It can take a few minutes for the Contour Operator to become available.
```

Comment:
Here you have _the Contour Operator_. Should these references be consistent?

Reply:
Updated!